online-shop/src/app/classes.ts

```diff
@@ -0,0 +1,18 @@
+export interface Product{
+id:String;
+name:String;
+category:String;
+price:Number;
+image?:String;
+description?:String;
+quantity:number;
+}
+export interface ProductPostTemplate{
```

Comment:
this doesn't seem used, it only appears in an import. do you still need it?
please also clean up all other unused imports from all files.